sonarqube/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM sonarqube
 
+RUN apk add --no-cache curl
```

Comment:
Given it's based on Alpine and thus has Busybox's `wget` built-in, wouldn't it be simpler to port this healthcheck to use `wget`? (as in the suggestion below)

```suggestion
```